tests/integration/100.command.dev.test.cjs

```diff
@@ -136,9 +136,11 @@ test('background function clientContext,identity should be null', async (t) => {
       })
       .buildAsync()
 
-    await withDevServer({ cwd: builder.directory }, async ({ outputBuffer, url }) => {
+    await withDevServer({ cwd: builder.directory }, async ({ outputBuffer, url, waitForLogMatching }) => {
```

Comment:
What is this about?

Reply:
Daniel added this in https://github.com/netlify/cli/commit/6c1a660990887af4c562bcdfba7328de9fd48e98, but i'm also unsure what it's about. Presumably it fixes some race condition?